chminfo_service/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015, wootski (https://github.com/wootski).
```

Comment:
I really hate to be a stickler here, but I'm not sure we can accept code that has what I believe to be an unenforceable copyright statement. Would you be comfortable using your real name here? I'm happy to be overridden my @mgoffin if he says we should just accept it but my experience indicates it is important to have proper attribution of copyright in case anything comes up in the future.